youtube_dl_gui/locale/ar_SA/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -196,200 +191,187 @@ msgstr "إيقاف تشغيل النظام"
 
 #: youtube_dl_gui/mainframe.py:141
 msgid "Extension"
-msgstr ""
+msgstr "نوع الملف (الصيغة)"
 
 #: youtube_dl_gui/mainframe.py:142
 msgid "Size"
 msgstr "الحجم"
 
 #: youtube_dl_gui/mainframe.py:143
 msgid "Percent"
-msgstr "النِسبة"
+msgstr "النِسبة المئوية"
 
 #: youtube_dl_gui/mainframe.py:144
 msgid "ETA"
 msgstr "الوقت المتبقي"
 
 #: youtube_dl_gui/mainframe.py:145
 msgid "Speed"
-msgstr "السرعة"
+msgstr "سرعة التنزيل"
 
 #: youtube_dl_gui/mainframe.py:146
 msgid "Status"
 msgstr "الحالة"
 
 #: youtube_dl_gui/mainframe.py:235
 msgid "Get URL"
-msgstr ""
+msgstr "جد الرابط"
 
 #: youtube_dl_gui/mainframe.py:236
-#, fuzzy
 msgid "Get command"
-msgstr "أوامر"
+msgstr "جد أمر"
```

Comment:
Are you sure this is the correct translation?

"Get command" = "Copy the youtube-dl command line arguments to the clipboard"